third_party/nvidia/backend/compiler.py

```diff
@@ -303,15 +307,19 @@ def make_cubin(src, metadata, opt, capability):
                 if e.returncode == 255:
                     raise RuntimeError(f'Internal Triton PTX codegen error: \n{log}')
                 elif e.returncode == 128 + signal.SIGSEGV:
-                    raise RuntimeError(
-                        f'Please run `ptxas {fsrc.name}` to confirm that this is a bug in `ptxas`\n{log}')
+                    delete_tmp_files = False
+                    raise RuntimeError('`ptxas` segfaulted. Please run the repro command below to confirm.\n'
+                                       f'Repro command: {cmd}\n'
+                                       f'`ptxas` logs (possibly empty): see {flog.name}\n'
+                                       f'To clean up temporary files: rm {fsrc.name} {flog.name}\n')
                 else:
                     raise RuntimeError(f'`ptxas` failed with error code {e.returncode}: \n{log}')
```

Comment:
Do we also need the temp file on this path?

Reply:
I suppose it may be useful for all three paths -- what if we get rid of the `finally` and move that logic into the `try` block?

Reply:
Sounds good.